app/src/main/java/com/github/se/cyrcle/ui/report/AdminScreen.kt

```diff
@@ -472,16 +470,13 @@ fun AdminScreen(
                                               style = MaterialTheme.typography.titleMedium)
                                         },
                                         text = {
-                                          val selectedObject =
-                                              reportedObjectViewModel.selectedObject
-                                                  .collectAsState()
-                                                  .value
+                                          reportedObjectViewModel.selectedObject.collectAsState()
                                           Column {
                                             if (selectedObject != null) {
                                               ReportDetailsContent(
-                                                  selectedObject.objectType,
-                                                  selectedObject.objectUID,
-                                                  selectedObject.userUID,
+                                                  selectedObject!!.objectType,
```

Comment:
why do the !! re-become necessary? I specifically changed this yesterday especially because I enveloped it in a null check at @matttsss's request

Reply:
The IDE wouldn't let me use anything else. The null check is still here but you initialized `selectedObject` three times where it was not necessary so I removed everything to keep only the first declaration of the val. This way the IDE requested that I put !! here.

Reply:
Ah, okay. I see. Glad you found an even more efficient way to clean up the code!